examples/search.js

```diff
@@ -194,6 +194,11 @@ app.module.controller('AppSearchController', app.SearchController);
  * @constructor
  */
 app.MainController = function() {
+  proj4.defs('EPSG:21781',
```

Comment:
There are more places that redefine 21781, I will replace them.

But what about using stuff from the gmf module in a pure ngeo example?